alws/crud/build_node.py

```diff
@@ -471,15 +471,26 @@ async def __update_built_srpm_url(db: Session, build_task: models.BuildTask):
     )
     srpm_artifact = srpm_artifact.scalars().first()
 
-    # if SRPM isn't built in first arch of project,
-    # we need to stop building project
+    # if SRPM isn't built in first arch of project, we need to stop building
+    # the project and fast-fail the uncompleted tasks
     if (not srpm_artifact and build_task.status == BuildTaskStatus.FAILED
             and uncompleted_tasks_ids):
         update_query = update(models.BuildTask).where(
             models.BuildTask.id.in_(uncompleted_tasks_ids),
         ).values(status=BuildTaskStatus.FAILED)
```

Comment:
we can update "fast-fail log" here too, instead of additional update query

Reply:
Nice catch!